site/content/en/docs/Guides/access-api.md

```diff
@@ -134,6 +134,131 @@ kubectl logs create-gs-6wz86-7qsm5  --namespace agones-system
 ```
 You have just created a GameServer using Kubernetes Go Client.
 
+## What are Informers and Listers? How can I use these?
+
+Informers and Listers are mechanism for Kubernetes.
+Almost all of the native Kubernetes controllers and custom controllers use these mechanisms.
+Kubernetes's control plane can be accessed via Kubernetes APIs, however,
+the load for control plane cannot be ignored when the client program accesses to the control plane every time.
+
+Informers and Listers can reduce the load for the control plane by using the in-memory-cache.
+When Informes and Listers are created, the in-memory-cache called as the store are also created.
+The client program accesses and queries to the in-memory-cache, not control plane,
+which enables reducing the load.
+
+Informer's role and Lister's role are different.
+The Informer is the mechanism for watching Kubernetes object's event.
+When the Kubernetes objects changes(e.g. CREATE,UPDATE,DELETE), the `Event` object will create.
+The Informer lists and watches the object's changes through `Event` from the control plane.
+When the `Event` happens, the informer will watch it and store objects to the in-memory-cache.
+
+The Lister is the mechanism for listing Kubernetes object's.
+Users can list the Kubernetes objects by using Kubernetes APIs, however,
+if users use List API, it causes the load for the control plane.
+The Informer stores the cache of the object to the in-memory-cache.
+If users use the in-memory-cache, the load can be avoidable.
+The Lister provides the list method while using the in-memory-cache.
```

Comment:
```suggestion
The Lister is the mechanism for querying Kubernetes object's against the client side in-memory cache.
Users can list the Kubernetes objects by using Kubernetes APIs, however,
if users do so, it places load on the Kubernetes control plane.

Since an Lister stores objects in an in-memory cache, queries against a come at practically no cost.
```